test/utils/Defaults.sol

```diff
@@ -501,15 +502,30 @@ contract Defaults is Constants, Merkle {
         return LEAVES;
     }
 
+    function schedule() public pure returns (MerkleLL.Schedule memory schedule_) {
+        schedule_.startTime = STREAM_START_TIME;
+        schedule_.cliffDuration = CLIFF_DURATION;
+        schedule_.totalDuration = TOTAL_DURATION;
+    }
+
     function tranchesMerkleLT() public view returns (LockupTranched.Tranche[] memory tranches_) {
         tranches_ = new LockupTranched.Tranche[](2);
         tranches_[0] = LockupTranched.Tranche({ amount: 2500e18, timestamp: uint40(block.timestamp) + CLIFF_DURATION });
         tranches_[1] = LockupTranched.Tranche({ amount: 7500e18, timestamp: uint40(block.timestamp) + TOTAL_DURATION });
     }
 
     /// @dev Mirros the logic from {SablierMerkleLT._calculateTranches}.
-    function tranchesMerkleLT(uint128 totalAmount) public view returns (LockupTranched.Tranche[] memory tranches_) {
+    function tranchesMerkleLT(
+        uint40 startTime,
+        uint128 totalAmount
+    )
+        public
+        view
+        returns (LockupTranched.Tranche[] memory tranches_)
+    {
         tranches_ = tranchesMerkleLT();
+        tranches_[0].timestamp = startTime + CLIFF_DURATION;
```

Comment:
To mirror the logic, it should be 

```solidity
tranches_ = new LockupTranched.Tranche[](2);
if (streamStartTime == 0) {
    tranches_[0].timestamp = uint40(block.timestamp) + CLIFF_DURATION;
    tranches_[1].timestamp = uint40(block.timestamp) + TOTAL_DURATION;
} else {
    tranches_[0].timestamp = streamStartTime + CLIFF_DURATION;
    tranches_[1].timestamp = streamStartTime + TOTAL_DURATION;
}
```

Also the natspec should be `@dev Mirros the logic from {SablierMerkleLT._calculateStartTimeAndTranches}`

Addressed in https://github.com/sablier-labs/v2-core/pull/1027

Reply:
> mirror the logic, it should be

yeah, true, but this function was used only when `streamStartTime != 0` so it worked, but we can remove the default one(with no params) as you did in your PR

Reply:
Yeah. I also introduced a new test where `streamStartTime` could be 0.